tests/test_benchmark/task_grid.py

```diff
@@ -76,23 +82,36 @@ def dataset_transform(self):
 
 tiny_imagenet = TinyImageNet()
 renderedSST2 = RenderedSST2()
-winoground = Winoground()
+aro = AROFlickrOrder()
+oxford_pets = OxfordPetsClassification()
+voc2007 = VOC2007Classification()
+flickr = Flickr30kI2TRetrieval()
+roxford_mc = ROxfordEasyI2IMultiChoice()
+cvbench_count = CVBenchCount()
+sts16 = STS16VisualSTS()
 
 ## method override to speed up tests
 tiny_imagenet.dataset_transform = dataset_transform.__get__(tiny_imagenet)
 renderedSST2.dataset_transform = dataset_transform.__get__(renderedSST2)
-winoground.dataset_transform = dataset_transform.__get__(winoground)
+aro.dataset_transform = dataset_transform.__get__(aro)
```

Comment:
Not too familiar with this part of the test suite, but as I understand it you are essentially downsampling the datasets to 2 samples?

It might be worth renaming `dataset_transform` to `downsampled_dataset_transform` to make this clear.

Reply:
Comparing to the last PR in the MIEB branch, this PR added about 20s (to 13min18s for Python3.11), so for now I'd say it's not too significant. But yes, using MockTests can be a good next step! Let's create an issue.

And re: renaming, sure, will do.

Reply:
Let us - feel free to merge this

> 13min18s

Oh wow, that is a long test suite (might be worth profiling that down to a reasonable level)

Reply:
Yeah, the last PR on this branch was at 12 minutes ish.